bin/broker-server-start.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I'm a bit concerned about having two scripts for starting Kafka in a customer environment:
* The one that appears in the quickstart documentation (kafka-server-start)
* The one we want them to use (broker-server-start)

How about simply replacing the original script and having one script, with one well-documented name? It will still start Kafka, so users shouldn't mind.